### ISSUE TITLE
Can't create HDF

### ISSUE THREAD

**Issue by tukicat** [midwan/amiberry#269]
Hi, 

Thanks for you work on Amiberry, the new version is great. 

I am using Amiberry 2.14 on the my rpi3. However, I can't create a .hdf file.

I select create a hdf and give it a file name, it then says cannot create .hdf.

Any ideas what is causing this?

**Comment by midwan** [comment]
@tukicat 
It could be lack of permissions to write to the location you selected?

Which distro are you trying this with?
Which path did you choose to save the HDF in?
Did you try a path where you know you have access to, e.g. in your home directory?

**Comment by tukicat** [comment]
@midwan 

Hi, I suspected it might be due to permissions, but I haven't been able work out why.

I am using the Retropie distro, on my rpi3.

I have tried writing to two locations, these are: -

opt/retropie/emulators/Amiberry
pi/RetroPie/roms/amiga 

I'll try my home directory when I'm back later today.

**Comment by tukicat** [comment]
Hi, I can confirm this is a permissions issue. 

I tried creating the .hdf in the home directory and it did not work. I then remembered that I have created a savestate before and that worked fine. So I tried to create the .hdf in the savestates directory, on the basis that Amiberry has written to this directory before. 

This worked fine and it wrote the .hdf.

The path for my savestates is the default one /opt/retropie/configs/amiga/amiberry/savestates.

This also explains another issue I was having. I decided to create a .hdf in WinUAE and use it with Amiberry. It worked fine but it would not let me write to the .hdf. I think this was because it was in a directory that I could not write to.
